### PR TITLE
Issue/3726 UI for loading state of the collection payments dialog

### PR DESCRIPTION
**Pull request by kidinov**
Parent issue #3726

### Design
vaw3DvewbUwuQnPXpdNGGH-fi-178%3A993060

Fonts and are subject of another change

Another change here is that payment dialogs (collection and connection) are not cancelable anymore by clicking outside of the dialog itself

### How to test
Orders -> Order -> click collect payment button

https://user-images.githubusercontent.com/4923871/118483832-2d7d7d00-b71f-11eb-98fb-32771d47fbfc.mp4

### CODE REVIEW

File: WooCommerce/src/main/res/layout/fragment_card_reader_payment.xml [hunk 1]
```diff
@@ -49,30 +40,43 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/margin_extra_extra_medium_large"
         android:contentDescription="@null"
-        app:layout_constraintBottom_toTopOf="@id/stub"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/amount_label"
         tools:srcCompat="@drawable/ic_card_reader" />
 
-    <ViewStub
```
Comment (kidinov) [inline WooCommerce/src/main/res/layout/fragment_card_reader_payment.xml]: It was unused so I removed it. Is this ok or I'm missing something?

Comment (malinajirka) [reply WooCommerce/src/main/res/layout/fragment_card_reader_payment.xml]: Good call, it was used before we replaced ConstraintLayout with LinearLayout and I forgot to remove it ;).

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -34,15 +34,6 @@
           <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
         </value>
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```
Comment (malinajirka) [inline .idea/codeStyles/Project.xml:37]: Is this change intentional?

**Review by malinajirka — APPROVED** [review]
Great job, thanks!
